projects/vdk-heartbeat/src/vdk/internal/heartbeat/hearbeat.py

```diff
@@ -28,6 +28,12 @@ def __init__(self, config: Config):
     @TestDecorator()
     def run(self):
         run_test = create_test_instance(self._config)
+        if self._config.bare_test:
+            self._run_bare_test(run_test)
```

Comment:
Maybe we can call it a base test and the other one can be a control plane test. I was having trouble picking the name as well, so suggestions are welcome. I was thinking of doing a decorator pattern, but figured it might be too complex.